lib/conditions/predefined.js

```diff
@@ -122,5 +122,21 @@ module.exports = [
       },
       required: ['expression']
     }
+  },
+  {
+    name: 'authenticated',
+    handler: (req, conditionConfig) => req.isAuthenticated(),
+    schema: {
+      type: 'object',
+      properties: {}
+    }
+  },
+  {
+    name: 'unauthenticated',
```

Comment:
This is the equivalent of:

```json
{
  "name": "not",
  "condition": {
    "name": "authenticated"
  }
}
```

Right?

If so, does it make sense to have both?  Looking for options.  `authenticated` and `unauthenticated` look very similar.  Is there another word we can use instead of `unauthenticated`?  Maybe `anonymous`?

Reply:
correct. you can do `not`, but is it convenient ? 
`authenticated` vs  `anonymous`works for me, but I want to keep both